packages/shared/src/lib/geolocation.ts

```diff
@@ -1,11 +1,18 @@
-export type CountryGroupId =
-    | 'GBPCountries'
-    | 'UnitedStates'
-    | 'AUDCountries'
-    | 'EURCountries'
-    | 'International'
-    | 'NZDCountries'
-    | 'Canada';
+import * as z from 'zod';
```

Comment:
I've heard differing opinions of using wildcards (*) in imports - are we ok with importing a wildcard instead of being more specific?

Reply:
Great question...can't remember the details but not using the wildcard seemed to break it? @tomrf1 maybe you recall

Reply:
we had mysterious runtime errors when importing the other way -
https://github.com/guardian/support-dotcom-components/pull/1053/commits/6547348113888f9d85241ef7029d9e45dd90dcc5